android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/UnityMessage.java

```diff
@@ -0,0 +1,34 @@
+package com.shopify.unity.buy;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.UUID;
+
+public class UnityMessage implements JsonSerializable {
+
+    public static String CONTENT_KEY = "Content";
+    public static String IDENTIFIER_KEY = "Identifier";
+
+    public String content;
```

Comment:
I think you can make it immutable as they are initialized only in constructor (make them final)